server/controllers/users.js

```diff
@@ -280,7 +280,7 @@ export function volunteerCallback(req, res) {
 
 export function contactCallback(req, res) {
   const { call_id: id } = req.params;
-  const { call_sid } = req.body;
+  const { DialCallSid: call_sid } = req.body;
```

Comment:
Good catch.

Reply:
Is there a test to be updated for this? I feel like there should be?

Reply:
Yes, this will be in the callback endpoint tests... I think Duy had that ticket, but needed to wait until we connected it to see what the response obj would be.